app/models/institution.ts

```diff
@@ -21,9 +22,22 @@ export default class Institution extends OsfModel {
     @attr('fixstring') description!: string;
     @attr('string') logoPath!: string;
     @attr('string') authUrl!: string;
+    @attr('object') assets!: any;
     @hasMany('user', { inverse: 'institutions' }) users!: DS.PromiseManyArray<User>;
     @hasMany('node', { inverse: 'affiliatedInstitutions' }) nodes!: DS.PromiseManyArray<Node>;
     @hasMany('registration', { inverse: 'affiliatedInstitutions' }) registrations!: DS.PromiseManyArray<Registration>;
+
+    @computed('assets', 'id')
+    get logoUrl(this: Institution): string {
+        let assetsUrl = '';
+        if (this.assets !== null) {
+            const { logo } = this.assets;
+            assetsUrl = logo;
+        } else {
+            assetsUrl = `/static/img/institutions/shields-rounded-corners/${this.id}-shield-rounded-corners.png`;
```

Comment:
Do we have cases where the logo asset is not set on the institutions, but we DO have an asset for it? If so, do we want to assume that we should use the asset?

Reply:
Oh, wait, is this to support both static assets and CDN assets?

Reply:
I think this is preparing for the future. Right now, the assets are being stored in a set location, but I think we wanted to do this so that, when institutions becomes a `provider` like everything else in collections, and the assets are stored the same way everything else is, this just works.